### PR TITLE
Minor documentation strings in each stack file.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
One-liners saying what's in each file.

Looking for a review just to make sure the sentences make sense.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: infra/database.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: ''
+Description: 'The database - including the networking required to get it to interface with the VPC'
```
Comment (carolinedanzi) [inline infra/database.yaml:2]: suggestion: I am not sure what "VPC" means, mind expanding the acronym?

File: infra/monitoring.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: ''
+Description: 'Monitoring for the UAT - see comments in file for alerts.'
```
Comment (carolinedanzi) [inline infra/monitoring.yaml:2]: suggestion (blocking): Can we specify which file?

File: infra/network.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: ''
+Description: 'The VPC for the UAT.  One region, to save costs, but two AZs for reliability.  Parallel stacks - two public and two private subnets.'
```
Comment (carolinedanzi) [inline infra/network.yaml:2]: suggestion (blocking): Expand AZ and VPC acronyms for readability (aka I do not know what they mean :) )

**Review by carolinedanzi — APPROVED** [review]
Thank you for adding descriptions, these are super helpful! A few minor suggestions for readability but I think this looks great

**Review by natsid — APPROVED** [review]
Thanks Nathan! Agree with Caroline's suggestions.

File: infra/network.yaml (hunk 3 above)
Comment (ndmckinley) [reply infra/network.yaml]: Done!  Availability Zone and Virtual Private Cloud.

File: infra/monitoring.yaml (hunk 2 above)
Comment (ndmckinley) [reply infra/monitoring.yaml]: Done - this file! :)

File: infra/database.yaml (hunk 1 above)
Comment (ndmckinley) [reply infra/database.yaml]: Sure!  It's amazon's word for "network".  Stands for "virtual private cloud", by analogy to VPN.  It's just a buncha private IPs, subnets and route tables.  Replaced with "the private network".

**Comment by ndmckinley** [comment]
Failure screenshot:
image

Nothing to worry about, this'll be my last PR from a fork.  Don't want to move it for just a docs change - I'll run the checker locally.

**Comment by ndmckinley** [comment]
Cool, yep, passes locally, merging...

**Pull request merged**